electron/config.ts

```diff
@@ -321,6 +321,8 @@ class GlobalConfigV0 extends GlobalConfig {
     const defaultWine = isWindows ? {} : await this.getDefaultWine()
 
     return {
+      checkUpdatesInterval: 10,
+      enableUpdates: true,
```

Comment:
I saw that you did add the option to the config file but haven't on the frontend for the user to change it.

Reply:
I will add that in a second PR. I don't want to make this one too big, or to get many conflicts.